feature-libs/product-configurator/rulebased/cpq/occ/model/occ-cpq-configurator-endpoints.model.ts

```diff
@@ -59,5 +63,9 @@ declare module '@spartacus/core' {
      * Endpoint for reading a CPQ configuration attached to a given order entry over OCC
      */
     readCpqConfigurationForOrderEntryFull?: string | OccEndpoint;
+    /**
+     * Endpoint for reading a CPQ configuration attached to a given quote entry over OCC
+     */
+    readCpqConfigurationForQuoteEntryFull?: string | OccEndpoint;
```

Comment:
I thought both methods `readCpqConfigurationForQuoteEntry` and `readCpqConfigurationForQuoteEntryFull` will be called over OCC.
Please, write a clearer method comment for  `readCpqConfigurationForQuoteEntryFull`.
Otherwise, one thinks that only the second method is called over OCC.

Reply:
@ChristophHi, could you have a look?

Reply:
readCpqConfigurationForQuoteEntry no longer present, comment adjusted